src/sparseml/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -112,16 +119,31 @@ def save_pretrained_wrapper(
             # state_dict gets passed in as a kwarg for FSDP models
             state_dict = kwargs.get("state_dict", None)
 
-            if qat_active(model):
+            if qat_active(model) or is_model_quantized(model):
                 _LOGGER.info(
```

Comment:
we should only throw a warning if model is quantized with the old quantization modifier 👍